nav2_rviz_plugins/src/nav2_panel.cpp

```diff
@@ -281,31 +331,58 @@ Nav2Panel::Nav2Panel(QWidget * parent)
   // State entered to reset the nav2 lifecycle nodes
   reset_ = new QState();
   reset_->setObjectName("reset");
+
   // State entered while the navigate_to_pose action is active
-  running_ = new QState();
-  running_->setObjectName("running");
-  running_->assignProperty(start_reset_button_, "text", "Cancel");
-  running_->assignProperty(start_reset_button_, "toolTip", cancel_msg);
+  running_nav_through_poses_ = new QState();
+  running_nav_through_poses_->setObjectName("running_nav_through_poses");
+  running_nav_through_poses_->assignProperty(start_reset_button_, "text", "Cancel");
+  running_nav_through_poses_->assignProperty(start_reset_button_, "toolTip", cancel_msg);
+
+  running_nav_through_poses_->assignProperty(pause_resume_button_, "text", "Pause");
+  running_nav_through_poses_->assignProperty(pause_resume_button_, "enabled", false);
+
+  running_nav_through_poses_->assignProperty(navigation_mode_button_, "text", "Waypoint mode");
+  running_nav_through_poses_->assignProperty(navigation_mode_button_, "enabled", false);
```

Comment:
This needs to go back to the original state.. `running_` state should be untouched..